src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6600,4 +6600,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_FunctionPointerTypesInAttributeNotSupported" xml:space="preserve">
     <value>Using a function pointer type in a 'typeof' in an attribute is not supported.</value>
   </data>
+  <data name="IDS_FeatureUsingGlobal" xml:space="preserve">
+    <value>global using directive</value>
```

Comment:
Should this have a comment for localization team?

Reply:
> Should this have a comment for localization team?

Historically we do not do this.

---
In reply to: [581753713](https://github.com/dotnet/roslyn/pull/51426#discussion_r581753713) [](ancestors = 581753713)

Reply:
@AlekseyTs There are *few* resources that add similar comments for loc team:

https://github.com/dotnet/roslyn/blob/2ddaf1c5aa13b062fb364b8dd7805d6a737afb59/src/Compilers/CSharp/Portable/CSharpResources.resx#L6519

This results in better translations. Was there a reason to not do this? (I see the resources I'm referring to are very very recent)

Reply:
> Was there a reason to not do this? 

In my opinion there was no a good reason to add those comments.

---
In reply to: [582073782](https://github.com/dotnet/roslyn/pull/51426#discussion_r582073782) [](ancestors = 582073782)